index.js

```diff
@@ -12,6 +12,7 @@ class ServerlessPlugin {
     this.serverless = serverless;
     this.provider = serverless.getProvider("aws");
     this.options = options;
+    this.resolvedLayers = [];
```

Comment:
I prefer `Set` over `Array` to store unique items.

```suggestion
    this.resolvedLayerSet = new Set();
```

Reply:
Done